SUMMARY.md

```diff
@@ -9,8 +9,9 @@
 - [🎓 Guides](guides/README.md)
   - [🚶 First Steps in Crypto](guides/newbie.md)
   - [💶 Get agEUR](guides/getAgEUR.md)
-  - [💸 Pay and get paid with agEUR](guides/ageur-power-usecase.md)
-  - [🏦 agEUR as a treasury asset](guides/ageur-treasury.md)
+  - [🧩  Use cases](/guides/use-cases/ageur-power-usecase.md)
+    - [💸 Pay and get paid with agEUR](/guides/use-cases/ageur-power-usecase.md)
```

Comment:
Here file root has been changed, and it'll break some references, make sure in other guides that are linking to this file, reference won't be broken